### DISCUSSION TITLE
I cant display a graph with plot command

### DISCUSSION THREAD

**Discussion by actnonymous** [google/lightweight_mmm#76] [General]
I cant display the plot in the example and matplotlib is installed correctly. Thanks for your answers
![Screenshot from 2022-09-24 16-28-09](https://user-images.githubusercontent.com/45433503/192119352-eb138dd3-cf9e-4b68-bf52-e1b282c8cde2.png)

**Comment by pabloduque0** [comment]
Hello @actnonymous !

If you are not running this from an Ipython env (colab, jupyter, ...) you might need to run `plt.show()` after. Not sure on what system you are but the figure of the plot is returned so you should be able to display it using the indicated call for your case.

If that is not your case, please provide a bit more info on how/where you are running that.

Hope it helps!

**Reply by actnonymous** [reply]
I used rmarkdown in Rstudio with a python enviroment. I followed your sugestion plt.show() but didnt work, then i tried with plot.matplotlib.pyplot.show and plot.plt.show() commands but didn't show the graph.

Then i changed my code into visual code with jupyter extension and did work.

Thanks.
